activerecord/test/cases/strict_loading_test.rb

```diff
@@ -98,6 +98,20 @@ def test_strict_loading_n_plus_one_only_mode_does_not_eager_load_child_associati
     end
   end
 
+  def test_default_mode_is_all
+    developer = Developer.first
+    assert_predicate developer, :strict_loading_all?
```

Comment:
```suggestion
    assert_predicate(developer, :strict_loading_all?)
```

We don't enforce this on rubocop because it would result in a large cosmetic change which we try to avoid. I think an exception was made to enforce double quotes. There is also no `.rubocop_todo.yml` to enforce this progressively, so we're left with solving this in code review. :bow:

Reply:
I just realized this is the exact _opposite_ style suggestion made earlier: https://github.com/rails/rails/pull/51339#discussion_r1528676022

I accepted yours, which was my original style.

Reply:
Actually, ugh. L112 also has the parenthetical-less style which was suggested: https://github.com/rails/rails/pull/51339/files#diff-0f8c8b854a62005e61f7855cf989a0be4836f591cef662baddff17194642f55fR112

I've gone back to removing the parens here, too. It's two previous commenters making this suggestion, so 🤷🏻‍♂️

Reply:
Sorry I may have misread that, it was early :see_no_evil: 

My impression was that some maintainers actively avoid using Seattle.rb style (no-parens), but let's wait for another look.

Reply:
fwiw this entire file is paren-less 🤷🏻‍♂️